src/openMVG/sfm/pipelines/global/GlobalSfM_rotation_averaging.cpp

```diff
@@ -53,9 +53,12 @@ bool GlobalSfM_Rotation_AveragingSolver::Run(
         return false;
       KeepOnlyReferencedElement(set_remainingIds, relativeRotations);
     }
-    break;
+	break;
+    default:
+    std::cerr << "Unknown relative rotation inference method: " 
+            << (int) eRelativeRotationInferenceMethod << std::endl;
```

Comment:
Thanks for the suggestion.
Note that bSuccess will be false and false will be return later